executor/aggfuncs/func_value.go

```diff
@@ -114,10 +138,9 @@ type value4String struct {
 	isNull bool
 }
 
-func (v *value4String) evaluateRow(ctx sessionctx.Context, expr expression.Expression, row chunk.Row) error {
-	var err error
+func (v *value4String) evaluateRow(ctx sessionctx.Context, expr expression.Expression, row chunk.Row) (memDelta int64, err error) {
 	v.val, v.isNull, err = expr.EvalString(ctx, row)
-	return err
+	return int64(len(v.val)), err
```

Comment:
`memDelta` should be the currently `v.val` used memory minus the previously `v.val` used memory.

Reply:
Ah, yes, you are right. I'll fix it now.